ksql-common/src/main/java/io/confluent/ksql/util/SchemaUtil.java

```diff
@@ -239,4 +239,100 @@ public static synchronized String getSchemaDefinitionString(Schema schema) {
     return stringBuilder.toString();
   }
 
+  public static String getSQLTypeName(Schema schema) {
+    switch (schema.type()) {
+      case INT32:
+        return "INT";
+      case INT64:
+        return "BIGINT";
+      case FLOAT32:
+      case FLOAT64:
+        return "DOUBLE";
+      case BOOLEAN:
+        return "BOOLEAN";
+      case STRING:
+        return "VARCHAR";
+      case ARRAY:
+        return "ARRAY<" + getSQLTypeName(schema.valueSchema()) + ">";
+      case MAP:
+        return "MAP<" + getSQLTypeName(schema.keySchema()) + "," + getSQLTypeName(schema
+                                                                                       .valueSchema()) + ">";
+      default:
+        throw new KsqlException(String.format("Invalid type in schema: %s.", schema.toString()));
+    }
+  }
+
+  public static  synchronized String buildAvroSchema(final Schema schema, String name) {
+    StringBuilder stringBuilder = new StringBuilder("{\n\t\"namespace\": \"ksql\",\n");
+    stringBuilder.append("\t\"name\": \"" + name + "\",\n");
+    stringBuilder.append("\t\"type\": \"record\",\n");
+    stringBuilder.append("\t\"fields\": [\n");
+    boolean addCamma = false;
```

Comment:
nit: should be `addComma`

Reply:
this method was replaced as you suggested above.